### PR TITLE
Wait until app is installed to print notes

### PR DESCRIPTION
**Pull request by erikwilson**
Fixes cli timeout when installing app without a NOTES.txt file.

For rancher/rancher/issues/16201

### CODE REVIEW

File: cmd/app.go [hunk 1]
```diff
@@ -601,18 +601,29 @@ func templateInstall(ctx *cli.Context) error {
...
-			return errors.New("timed out waiting for app notes, the app could still be installing. Run 'rancher apps' to verify")
+			return errors.New("timed out waiting for app to install. Run 'rancher apps' to verify status")
 		}
 		timeout++
 		time.Sleep(2 * time.Second)
 		madeApp, err = c.ProjectClient.App.ByID(madeApp.ID)
 		if err != nil {
 			return err
 		}
+		for _, condition := range madeApp.Conditions {
```
Comment (dramich) [inline cmd/app.go]: Use the condition helpers for this and drop the loop.

File: cmd/app.go [hunk 2]
```diff
@@ -601,18 +601,29 @@ func templateInstall(ctx *cli.Context) error {
 		return err
 	}
 
-	// wait for the app's notes to be populated so we can print them
-	var timeout int
-	for len(madeApp.Notes) == 0 {
+	// wait for the app to be installed to print the notes
+	var (
+		timeout   int
+		installed bool
```
Comment (dramich) [inline cmd/app.go]: Not needed

File: cmd/app.go [hunk 3]
```diff
@@ -601,18 +601,29 @@ func templateInstall(ctx *cli.Context) error {
...
+	)
+	for !installed {
```
Comment (dramich) [inline cmd/app.go]: Use a naked for loop

**Comment by dramich** [comment]
LGTM

**Comment by cjellick** [comment]
lgtm

**Pull request merged**